lib/galaxy/webapps/galaxy/api/sanitize_allow.py

```diff
@@ -0,0 +1,84 @@
+"""
+API operations allowing clients to retrieve and modify the HTML sanitization allow list.
+"""
+import logging
+
+from galaxy import web
+from galaxy.webapps.base.controller import BaseAPIController
+
+log = logging.getLogger(__name__)
+
+
+class SanitizeAllowController(BaseAPIController):
+
+    @web.require_admin
+    @web.expose_api
+    def index(self, trans, **kwd):
+        """
+        GET /api/sanitize_allow
+        Return an object showing the current state of the toolbox and allow list.
+        """
+        return {'status': 'done', 'message': 'Tool allow list loaded.', 'data': self._generate_allowlist(trans)}
```

Comment:
```suggestion
        return self._generate_allowlist(trans)
```
I feel like status and message don't really add anything in these routes ?